addons/website_forum/views/website_forum.xml

```diff
@@ -142,129 +126,212 @@
+                        <a class="nav-link" t-attf-href="/forum/#{slug(forum)}/validation_queue">
+                            <i class="fa fa-check-square-o fa-fw text-warning"/>
+                            <b t-esc="forum.count_posts_waiting_validation" class="text-800"/>
+                        </a>
+                    </li>
+                    <li t-if="forum.count_flagged_posts" class="nav-item ml-2">
+                        <a class="nav-link" t-attf-href="/forum/#{slug(forum)}/flagged_queue">
+                            <i class="fa fa-flag fa-fw text-danger"/>
+                            <b t-esc="forum.count_flagged_posts" class="text-800"/>
+                        </a>
+                    </li>
+                </t>
+                <!-- Mobile 'Search Box' toggler-->
+                <li class="nav-item ml-4">
+                    <a data-toggle="collapse" href="#o_wforum_search" class="nav-link"><i class="fa fa-search"/></a>
+                </li>
+            </ul>
+
+            <!-- 'Search Box' -->
+            <form id="o_wforum_search" class="form-inline collapse w-100 w-md-auto pt-2 pt-md-0 d-md-flex"
+                  role="search" t-attf-action="/forum/#{ slug(forum) }" method="get">
+                <t t-call="website.website_search_box">
+                    <t t-set="_classes">w-100</t>
+                </t>
+            </form>
+        </div>
+    </div>
 </template>
 
 <!-- Template use to edit the welcome_message -->
 <template id="edit_welcome_message">
     <t t-call="website.layout">
-        <div t-field="forum.welcome_message"/>
+        <section t-attf-class="s_cover parallax s_parallax_is_fixed py-3 #{'bg-black-50' if forum.image_1920 else 'o_wforum_forum_card_bg text-white'}" data-scroll-background-ratio="1">
+            <span t-if="forum.image_1920" class="s_parallax_bg oe_img_bg oe_custom_bg" t-attf-style="background-image: url('#{website.image_url(forum, 'image_1920')}'); background-position: center;"/>
+            <div class="container">
+                <div class="row s_nb_column_fixed">
+                    <div class="col-lg-12 s_title" data-name="Title">
+                        <h1 class="s_title_thin o_default_snippet_text text-center"><t t-esc="forum.name"></t></h1>
+                    </div>
+                </div>
+                <div  class="forum_intro">
+                    <div t-field="forum.welcome_message"/>
+                </div>
+            </div>
+        </section>
     </t>
 </template>
 
 <!-- Display a post -->
 <template id="display_post_question_block">
-    <div t-if="question.has_validated_answer" class="validate_answer">
-        <div class="bg-success"></div>
-        <span title="Question has accepted answer" role="img" aria-label="Question has accepted answer"><i class="fa fa-check-circle text-success bg-white"/></span>
+    <div class="o_wforum_index_entry_title">
+        <div t-attf-class="d-inline-block mb-0 #{'h6' if len(question.name) > 80 else 'h5'}">
```

Comment:
@stefanorigano Somehow question.name can be False here so the template crashes (occurs when I go to a post then click on the author avatar...). Same for another same condition below.

Reply:
btw, it is strange to have font size that change from one question to another one